### PR TITLE
Update 'paradox--package-repo-list' even if the repos hasn't star

### PR DESCRIPTION
**Pull request by PierreTechoueyres**
Decorrelate update of 'paradox--package-repo-list' from counting stars.

### CODE REVIEW

File: helpers/paradox-counter.el [hunk 1]
```diff
@@ -102,12 +102,11 @@ Also saves result to `package-star-count'"
...
-                   (if (numberp count)
-                       (progn
-                         (puthash name count paradox--star-count)
-                         (puthash name .repo paradox--package-repo-list))
-                     (paradox-log "FAILED: %s / %s" i name))))
+                 (progn
+		   (let ((count (paradox-fetch-star-count .repo)))
+		     (when (numberp count)
+		       (puthash name count paradox--star-count)))
+		   (puthash name .repo paradox--package-repo-list)))
```
Comment (Malabarba) [inline helpers/paradox-counter.el:109]: Looks like the indentation is a bit off here because you're using tabs. Could you switch to spaces?

**Review by Malabarba — CHANGES_REQUESTED** [review]

**Comment by Malabarba** [comment]
Hey there, thanks for the PR.
I'm curious though, are you using this file?
This file is meant as a way for me to populate the stars data cache that paradox uses (which, admittedly, I haven't done in a while)

File: helpers/paradox-counter.el (hunk 1 above)
Comment (PierreTechoueyres) [reply helpers/paradox-counter.el]: Hope the last commit fix this.

**Comment by PierreTechoueyres** [comment]
May be I'm wrong, but it seem to me this file is used to check the repo is from github before allowing you to retrieve commit list.

**Comment by PierreTechoueyres** [comment]
I think this will fix #176

**Comment by PierreTechoueyres** [comment]
ping ?

**Pull request closed without merging**